main.py

```diff
@@ -1,40 +1,96 @@
 import json
 import datetime
 import os
+import threading
+import time
+#from win10toast import ToastNotifier
+
+Hideout = ["tmp", "WorkBench", "MedStation", "Lavatory", "Scav Case", "Booze Generator", "Water Collector", "Nutrition Unit", "Intelligence Center"]
 
 isAlarm = True
 mainTmp = -1
 hideoutTmp = -1
 traderTmp = -1
 optionTmp = -1
-skillLevel = 0
+skillLevel = 1
+
 
 maker = []
+userDataSave = []
+userDataSave.append(maker)
+userDataSave.append(skillLevel)
+
+def getNowTime():
+    now_time = datetime.datetime.now()
+    
+    return now_time
+
+def toastAlarm():
+    while True:
+        for i in range(len(maker)):
+            if(getNowTime() > maker[i]):
+                maker[i] = datetime.datetime(2099, 12, 31, 0, 0, 0)
+                saveJsonData()
+                #toaster = ToastNotifier()
+                #toaster.show_toast(f'{Hideout[i]} is done!!', duration=10)
+
+                print(f'{Hideout[i]} is done!!')
+
+        time.sleep(5)
+
+def saveJsonData():
+    f = open('save.json', 'w')
+    json_data = {}
+    time_tmp = []
+    for i in maker:
+        time_tmp.append(i.strftime('%Y-%m-%d-%H-%M-%S'))
+    
+    json_data['time'] = time_tmp
+    json_data['level'] = skillLevel
 
-def saveJsonTime():
-    f = open('time.json', 'w')
-    json.dump(maker, f)
+    json.dump(json_data, f)
     f.close()
 
-def initStartProgram(): 
-    for _ in range(9):
-        maker.append(datetime.datetime.today())
-    
-    if(os.path.exists('time.json')):
-        f = open('time.json', 'r')
-        temp = json.load(f)
+def openUserData():
```

Comment:
```suggestion
def load_user_data():
```

`open` 보다는 `load`가 더 적합하지 않을까 싶습니다.